{
    "title": "The dnslib package through 0.9.16 for Python does not verify that the ID value in a DNS reply matches an ID value in a query.",
    "score": 5.3,
    "severity": "MEDIUM",
    "cwe_id": "NVD-CWE-noinfo",
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N",
    "cpe_match": [
        "cpe:2.3:a:dnslib_project:dnslib:*:*:*:*:*:python:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1007",
            "confidence": "40.8",
            "techniqueName": "System Service Discovery"
        },
        {
            "techniqueId": "T1033",
            "confidence": "25.4",
            "techniqueName": "System Owner/User Discovery"
        },
        {
            "techniqueId": "T1055",
            "confidence": "2.3",
            "techniqueName": "Process Injection"
        },
        {
            "techniqueId": "T1082",
            "confidence": "16.6",
            "techniqueName": "System Information Discovery"
        },
        {
            "techniqueId": "T1573",
            "confidence": "5.5",
            "techniqueName": "Encrypted Channel"
        }
    ]
}
